WordPress/Tests/VIP/PostsPerPageUnitTest.inc

```diff
@@ -1,8 +1,11 @@
 <?php
-
+// @codingStandardsChangeSetting WordPress.VIP.PostsPerPage posts_per_page 200
```

Comment:
For this sniff to be tested properly:
1. The sniff needs to be tested with the default values, i.e. the way the unit test case file was originally.
2. New unit tests have to be added for testing that changing the property works.

To do this:
* revert the unit test case file to the way it was before.
* add the above line at the bottom
* add one or more test cases to test that changing the property has the desired effect.
* add a second line like the above below the property change test case to reset the property value to the default value.
* adjust the line numbers in the unit test file

This will look something like (to be added at the bottom of the original file):
```php
// @codingStandardsChangeSetting WordPress.VIP.PostsPerPage posts_per_page 50
 $query_args['posts_per_page'] = 50; // OK.
 $query_args['posts_per_page'] = 100; // Bad.
 $query_args['posts_per_page'] = 200; // Bad.
 $query_args['posts_per_page'] = 300; // Bad.
// @codingStandardsChangeSetting WordPress.VIP.PostsPerPage posts_per_page 200
 $query_args['posts_per_page'] = 50; // OK.
 $query_args['posts_per_page'] = 100; // OK.
 $query_args['posts_per_page'] = 200; // OK.
 $query_args['posts_per_page'] = 300; // Bad.
// @codingStandardsChangeSetting WordPress.VIP.PostsPerPage posts_per_page 100
```

Reply:
I corrected this, should be ok now. Changed both the .inc and .php unit test files.